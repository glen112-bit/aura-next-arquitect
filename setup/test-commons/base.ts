import type {Config} from '@jest/types';
import path from 'path'; 

// Sync object
const config: Config.InitialOptions = {
  testEnvironment: 'jsdom',
  verbose: false,
  transform: {
    '\\.[jt]sx?$': ['babel-jest', {
      configFile: path.resolve(__dirname, '.babelrc'),
    }],
  }
};
export default config;
