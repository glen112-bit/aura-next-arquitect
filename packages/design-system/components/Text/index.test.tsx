import { customRender } from "@alura/test-commons/react-testing-library";
import { Text } from "./index";

const render = customRender();

describe("<Text />", () => {
	it("renders h1 tag", () => {
		const { container } = render(<Text tag="h1">sample Text</Text>);

		expect(container).toMatchSnapshot();
	});
});
