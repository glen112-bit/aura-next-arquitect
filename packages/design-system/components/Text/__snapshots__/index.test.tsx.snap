// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Text /> renders h1 tag 1`] = `
<div>
  <h1
    class="Text__StyledText-sc-1ul0hnu-0 cimzHl"
  >
    sample Text
  </h1>
</div>
`;
